### PR TITLE
Sukrut Recommendations Command

### PR DESCRIPTION
**Pull request by Sukrut1881**
I have began working on my recommendations portion:

1) Added a Day class which extends Venuelist and has a list of todos with it. Further added a DayList class. Itinerary inherits DayList and has a start/end date along with a hotel location.

2) Moved Todo to be a part of "planning package"

3) Modified recommendations command : 
     - ex. "recommend orchard between 23/04/19 and 24/04/19" , will automatically give a list of locations for 1 day (using new itinerary class) 

4) This recommendations list can now be saved as it is (needs to be worked on)

5) Create newItinerary from scratch needs to be implemented.

6) Logic to choose locations around hotel needs to be written

7) Save as PDF needs to be done

### CODE REVIEW

File: src/main/java/duke/model/planning/Itinerary.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        long days = tempDateTime.until(endDate, ChronoUnit.DAYS);
+        return Integer.parseInt(String.valueOf(days));
+    }
+
+    /**
+     * Prints the itinerary list in entirety.
+     *
+     * @return The String which lists the itinerary in full
+     */
+    public StringBuilder printItinerary() {
```
Comment (Jefferson111) [inline src/main/java/duke/model/planning/Itinerary.java]: Could possibly return a string here

File: src/main/java/duke/model/planning/Day.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package duke.model.planning;
+
+import duke.model.lists.VenueList;
+import duke.model.locations.Venue;
+
+import java.util.List;
+
+public class Day extends VenueList {
```
Comment (Jefferson111) [inline src/main/java/duke/model/planning/Day.java]: Day still sounds weird

File: src/main/java/duke/logic/parsers/ParserUtil.java [hunk 3]
```diff
@@ -48,6 +51,29 @@ protected static Event createEvent(String userInput) throws DukeException {
         return new Event(withinDetails[0].strip(), start, end);
     }
 
+    /**
+     * Parses the userInput and return a new Itinerary constructed from it.
+     *
+     * @param userInput The userInput read by the user interface.
+     * @return The new Itinerary object.
+     */
+    protected static Itinerary createRecommendation(String userInput) throws DukeException {
```
Comment (Jefferson111) [inline src/main/java/duke/logic/parsers/ParserUtil.java]: Magic numbers

File: src/main/java/duke/logic/parsers/ParserStorageUtil.java [hunk 4]
```diff
@@ -145,4 +147,44 @@ public static TrainStation createTrainFromStorage(String line) {
...
+        }
+        return todoList;
+    }
+
+    /**
+     * Returns the Day object in a string format.
+     *
+     * @return The Information for one day in a string format
+     */
+    public static String toDayString(Day day) {
```
Comment (Jefferson111) [inline src/main/java/duke/logic/parsers/ParserStorageUtil.java]: Is it possible to override the toString method?

File: src/main/java/duke/model/planning/Day.java (hunk 2 above)
Comment (Sukrut1881) [reply src/main/java/duke/model/planning/Day.java]: Changed to Agenda

File: src/main/java/duke/logic/parsers/ParserUtil.java (hunk 3 above)
Comment (Sukrut1881) [reply src/main/java/duke/logic/parsers/ParserUtil.java]: @hongchuan97

File: src/main/java/duke/logic/parsers/ParserStorageUtil.java (hunk 4 above)
Comment (Sukrut1881) [reply src/main/java/duke/logic/parsers/ParserStorageUtil.java]: Yes , overdid the to string method.

**Pull request merged**